app/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
         minSdkVersion 15
         targetSdkVersion 23
         versionCode 80
-        versionName "0.8.0"
+        versionName "0.9.0"
```

Comment:
Also we cannot go to 0.9.0 untill we solve all these 
https://github.com/fossasia/open-event-android/milestones/v0.9.0